caravel/translations/ru/LC_MESSAGES/messages.po

```diff
@@ -1809,39 +1809,39 @@ msgstr "Добро пожаловать!"
 msgid "Test Connection"
```

Comment:
I've commented out all the changes below, it still doesn't. Now I've made another changes, hope it will work.